Garmin.Connect/GarminConnectClient.cs

```diff
@@ -284,4 +285,28 @@ public Task<GarminGear[]> GetActivityGears(long activityId)
 
         return _context.GetAndDeserialize<GarminGear[]>(activityGearsUrl);
     }
+
+    public Task SetUserWeight(double weight)
+    {
+        dynamic garminUserSettings = new ExpandoObject();
+        garminUserSettings.userData = new ExpandoObject();
+        garminUserSettings.userData.weight = weight;
+        return _context.Put(UserSettingsUrl, garminUserSettings);
+    }
+
+    public Task SetUserSleepTimes(long? sleepTime, long? wakeTime)
+    {
+        dynamic garminUserSettings = new ExpandoObject();
```

Comment:
@HSyr Please see this comment, after the correction we will merge and I will put you as a reviewer

Reply:
Thanks for the reminder.
I still prefer dynamic to specific classes/records. For the current two SetXY functions, one would have to create 2x2 such classes used only in them while dynamic does not reduce the visibility of which properties are being set. Yes, dynamic is less efficient, but this this small loss I prefer over simplicity.

Reply:
Yes, I understand that the code is smaller and more concise, but this can create more problems for other contributors to read, and if you rename a field, you will have to look for these places. 
In this library, I just want to move away from `dynamic` use in favor of explicit immutable models.

If you have little time and you do not mind, I can correct to `record` models